.rspec

```diff
@@ -0,0 +1,2 @@
+--color
+--require spec_helper
```

Comment:
I could get behind the implicit `--require spec_helper`, but I'm not sure I want to force `--color` on everyone, sounds like something to put in `~/.rspec` if that's how you roll.